Base.rte/AI/NativeHumanAI.lua

```diff
@@ -52,7 +52,8 @@ function NativeHumanAI:Create(Owner)
 	-- the native AI assume the jetpack cannot be destroyed
 	if Owner.Jetpack then
 		if not Members.isPlayerOwned then
-			Owner.Jetpack.Throttle = 0.15	-- increase jetpack strength slightly to compensate for AI ineptitude
+			-- increase jetpack strength slightly to compensate for AI ineptitude
+			Owner.Jetpack.Throttle = 0.25
```

Comment:
I wonder how much this change is necessary, and how much it affects gameplay? I.e. do AI units often end up zooming too high in the sky with this? I'm not against it if it works well, I just wanna be cautious :)

Also, does the `isPlayerOwned` check mean that this doesn't affect actors that are on a human player's team? I don't have an issue with this, I'm just curious.

Reply:
I've not seen AI units zooming too high; the pathing doesn't really let them do that anyways, they'll stop when they feel they're high enough. As the AI doesn't really understand that it can use momentum to help, it's good that it has a bit of help, as it tends to stop and hover in place before moving left/right onto a ledge they're flying up to.

Yeah, isPlayerOwned check means it'll only apply to AI. That was already there before, I just upped the buff slightly

Reply:
Okay, glad to know about units zooming. We'll see if anybody has thoughts with the scripts you posted in #project-ai, this can always be tweaked as needed in future. Same thing with the non-human only check.

Feel free to resolve this